symfony/workspace/anyfiddle.json

```diff
@@ -0,0 +1,5 @@
+{
+  "openFiles": [],
```

Comment:
Can you add the default file for symfony to be opened in Anyfiddle here? What would the first file that would be edited by a user in symfony?

Reply:
Unfortunately, there used to have a default HomeController but it does not come with the skeleton anymore. Unless, I create one in the workspace and have this one opened, but I am afraid it could be too version dependent for the future.

Do you want to go with that solution then?

Reply:
I added a default controller and a default template view showing Hello World.

Reply:
Is there a README file that can be opened by default? Does that README have instructions to create controller.
Then we can show that.

The default controller is good but a user might be expecting to see the symfony default page.

Reply:
There is no README file that can be opened by default. Everything you need to pull to start is located on the symfony doc website, it is fairly extensive and well documented.

For example: https://symfony.com/doc/current/page_creation.html

I understand your concern and they used to have a default page but drove away from that, stating that the user would know what to do with it.

There is a "demo" project that can be created but last time I checked it requires more php libraries to be installed onto the system, and I am not sure having to deal with that extra overhead is what someone starting a new project wants.

Reply:
Ok. Let us keep the default controller but remove the route part so that the default symfony page will show. The openFiles in anyfiddle.json should be config/routes.yaml and src/Controller/DefaultController.php. The default routes.yaml have the route to the Default controller commented. So users can uncomment that and see hello world.

Reply:
I will merge once this is done